### DISCUSSION TITLE
Error installing in wsl

### DISCUSSION THREAD

**Discussion by SrinSS01** [NvChad/NvChad#1994] [General]
I tried installing nvimchas on wsl through this command `git clone https://github.com/NvChad/NvChad ~/.config/nvim --depth 1 && nvim` but got this error
```
Error detected while processing /home/srin/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/srin/.config/nvim/lua/core/init.lua:67: attempt to call local 'autocmd' (a nil value)
stack traceback:
        /home/srin/.config/nvim/lua/core/init.lua:67: in main chunk
        [C]: in function 'require'
        /home/srin/.config/nvim/init.lua:1: in main chunk
Press ENTER or type command to continue
```
How do I fix this?

**Comment by siduck** [comment]
@SrinSS01 please install neovim v 0.8.3 at least

**Reply by ItsMoMoHippo** [reply]
i am using nvim v0.9.5 and its still broken from me, im trying to migrate form windows nvchad to wsl nvchad

**Reply by siduck** [reply]
@ItsMoMoHippo  0.10 is the minimum required now
